src/app.rs

```diff
@@ -8,18 +8,10 @@ use std::collections::BTreeMap;
 use self::clap::{App, Arg, ArgGroup, AppSettings};
 use super::{Slurp, RunOptions, compile, DEFAULT_SHELL, maybe_s};
 
-macro_rules! warn {
-  ($($arg:tt)*) => {{
-    extern crate std;
-    use std::io::prelude::*;
-    let _ = writeln!(&mut std::io::stderr(), $($arg)*);
-  }};
-}
-
 macro_rules! die {
   ($($arg:tt)*) => {{
     extern crate std;
```

Comment:
Macros don't have lexical scope like functions, so even if something is in scope where they're defined, it might not be in scope where it's called.

Take that with a grain of salt though, I'm not sure if I was doing it correctly when I did that or just cargo-culting.